CONTRIBUTING.md

```diff
@@ -0,0 +1,412 @@
+   before creating an issue to avoid making a duplicate.
+2. Use the [correct template](https://github.com/knative/docs/issues/new) for
+   your new issue. There are two templates available:
+   - **Bug report**: If you're reporting an error in the existing documentation,
+     use this template. This could be anything from broken samples to typos.
+     When you create a bug report, include as many details as possible and
+     include suggested fixes to the issue. If you know which Knative component
+     your bug is related to, you can assign the appropriate
+     [Working Group Lead](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md).
+   - **Feature request**: For upcoming changes to the documentation or requests
+     for more information on a particular subject.
+
+Note that code issues should be filed against the
+[individual Knative repositories](http://github.com/knative), while
+documentation issues should go in the
+[`knative/docs`repo](https://github.com/knative/docs/issues). If the issue is
+specific to the [https://knative.dev](https://knative.dev) website, open the
+issue in the [`knative/website`repo](https://github.com/knative/website/issues).
+
+### Working group
+
+The [Knative Documentation Working Group](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md#documentation)
+meets weekly on Tuesdays and alternates between a 9am PT and a 4:30pm PT time to
+accommodate contributors in both the EMEA and APAC timezones.
+[Click here](https://calendar.google.com/calendar/embed?src=google.com_18un4fuh6rokqf8hmfftm5oqq4%40group.calendar.google.com)
+to see the exact dates on the Knative working group calendar. If you're
+interested in becoming more involved in Knative's documentation, start attending
+the working group meetings. You'll meet the writers currently steering our
+documentation efforts, who are happy to help you get started.
+
+### Getting started
+
+There are a couple different ways to jump in to the Knative doc set:
+
+- Look for a
+  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue)
+  in the backlog.
+
+- Try out Knative and send us feedback. For example, run through one of the
+  [install guides](docs/install/README.md) and then try
+  [Getting Started with Knative Serving](docs/serving/getting-started-knative-app.md).
+
+  You should keep a
+  [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
+  of your experience and then use that to open your first set of PRs. Examples:
+
+  - What was hard for you?
+  - Did you stumble on something because it wasn't clear?
+  - Was a dependency not mentioned?
+
+**A few pointers and other considerations**
+
+- Think about how you can improve the documentation as a whole. <br> For
+  example, how can you fix the issue you found so that others don't run into the
+  same challenges?
+
+- Are there multiple places that could be fixed?
```

Comment:
`that could be fixed` -> `where the issue could be fixed`?

Reply:
Yes. (Feel free to use "suggest" in GitHub if you want; I think the CLA-bot has finally sorted that out.)